pkgs/applications/misc/obsidian/default.nix

```diff
@@ -25,7 +25,7 @@ let
   filename = if stdenv.isDarwin then "Obsidian-${version}-universal.dmg" else "obsidian-${version}.tar.gz";
   src = fetchurl {
     url = "https://github.com/obsidianmd/obsidian-releases/releases/download/v${version}/${filename}";
-    sha256 = if stdenv.isDarwin then "1q9almr8k1i2wzksd09libgnvypj5k9j15y6cxg4rgnw32fa152n" else "sha256-Qz1Ic5FtxXIk8J/2spNZaqpPIgx3yNyXiAQllbVrGjw=";
+    sha256 = if stdenv.isDarwin then "1q9almr8k1i2wzksd09libgnvypj5k9j15y6cxg4rgnw32fa152n" else "sha256-H1Cl9SBz/mwN8gezFkcV4KxI7+xVjQV2AtyLKyjVpI8=";
```

Comment:
can you update the hash for darwin as well?

Reply:
Could someone who has push access to this PR set `sha256-KVWqdXzbQT93Dj4yesdnfFwAdYin1vfpPsj/ur1AGA0=` as the darwin hash? @ryantm, I'm guessing that's you.

Reply:
Hello there, I was going to manuall update Obsidian but used different hashes(?!) successfully. How is that possible? How did you obtain your hash? :smile: 
```

Reply:
thank you @bobby285271, that's #TIL for me